gnovm/pkg/gnolang/nodes.go

```diff
@@ -112,6 +112,8 @@ const (
 
 type Name string
 
+func (n Name) String() string { return string(n) }
```

Comment:
happy to remove this change if its adding unnecessary changes to the PR

Reply:
yes, it's not really necessary anyway